tests/integration.rs

```diff
@@ -518,7 +518,7 @@ fn multiple_paths() -> TestResult {
     assert_eq!(final_build_size, size_after_dry_run_clean);
 
     // Run a proper cargo-sweep ("clean") in the target directories
-    let mut args = vec!["-r", "--time", "0"];
+    let mut args = vec!["--time", "0"];
```

Comment:
Why this change?

Reply:
We only want to test the functionality for multiple paths, setting both the multiple paths and recursive could cause an error in either one so I just wanted to minimise the test case.